### ISSUE TITLE
"Stateof" and "2023" logo elements misplaced on Safari

### ISSUE THREAD

**Issue by pawelgrzybek** [Devographics/Monorepo#363]
"Stateof" and "2023" words misplaced on Safari

![html](https://github.com/Devographics/Monorepo/assets/3697104/42998ee6-6a0f-471b-878b-ecbeb31fcf82)

The solution is to remove `transform="rotate(30)"` from `<text>` elements, wrap them in a group `<g>` and apply translation on a group. Like so:

Before:
```
<text transform="rotate(30)" x="728" y="1260" font-size="100" text-anchor="end" class="0"><tspan>2</tspan><tspan>0</tspan><tspan>2</tspan><tspan>3</tspan></text>
```

After:
```
<g transform="rotate(30)">
    <text x="728" y="1260" font-size="100" text-anchor="end" class="0"><tspan>2</tspan><tspan>0</tspan><tspan>2</tspan><tspan>3</tspan></text>
</g>
```

CodePen: https://codepen.io/pawelgrzybek/pen/QWRbXRp

**Comment by chriskirknielsen** [comment]
Thank you! This should already be fixed with #365 and #366 but if you still see issues on Safari let me know!

**Comment by pawelgrzybek** [comment]
Yeah, this looks great now!
